app/views/landing_page/show.html.erb

```diff
@@ -1,13 +1,9 @@
 
 <div class="landing-page">
   <% @content_item.blocks.each do |block| %>
-    <% if block.full_width? %>
-      <%= render "landing_page/blocks/#{block.type}", block: %>
-    <% else %>
-      <div class="govuk-width-container">
-        <%= render "landing_page/blocks/#{block.type}", block: %>
-      </div>
-    <% end %>
+    <%= tag.div class: ["govuk-block__#{block.type}", ("govuk-width-container" unless block.full_width?)] do
```

Comment:
`["govuk-block__#{block.type}`
This is a great addition and something nice to test for.